src/app/app.component.ts

```diff
@@ -33,26 +36,45 @@ export class AppComponent implements OnDestroy, OnInit, AfterViewInit {
   windowWidth: number;
   sizeWarningOpen: boolean = false;
   minWidth: number = 1250;
+  activeAdminMessages$: any;
+  activeAdminMessagesQuery$: any;
+  activeAdminMessages: AdminMessage[];
+  adminMessageLevels = adminMessageLevels;
+  subscriptions$: Subscription[] = [];
+  get AdminMessageLevel(): typeof AdminMessageLevel {
+    return AdminMessageLevel;
+  }
 
   constructor(
     private notificationApolloService: NotificationApolloService,
     private organizationService: OrganizationApolloService,
     private configService: ConfigApolloService,
     private router: Router,
     private http: HttpClient,
-    private cfRef: ChangeDetectorRef
+    private cfRef: ChangeDetectorRef,
+    private projectApolloService: ProjectApolloService,
   ) { }
 
   ngOnInit(): void {
     this.initialRequests();
     NotificationService.subscribeToNotification(this, {
-      whitelist: ['notification_created', 'project_deleted', 'config_updated'],
+      whitelist: ['notification_created', 'project_deleted', 'config_updated', 'admin_message'],
       func: this.handleWebsocketNotification
     });
     this.initializeNotificationService();
     this.initWithConfigManager();
     this.checkBrowser();
-    UserManager.registerAfterInitActionOrRun(this, () => this.loggedInUser = UserManager.getUser(), true)
+    UserManager.registerAfterInitActionOrRun(this, () => this.loggedInUser = UserManager.getUser(), true);
+    [this.activeAdminMessagesQuery$, this.activeAdminMessages$] = this.projectApolloService.getAllActiveAdminMessages();
+    this.subscriptions$.push(this.activeAdminMessages$.subscribe((activeMessages: AdminMessage[]) => {
+      const saveActiveMessages = [];
+      activeMessages.forEach((message) => {
+        const saveMessage = { ...message };
```

Comment:
this is a shallow copy (not problematic in for this case) however i'd go with jsonCopy function in the future